frontend/src/utils/parse-collection-path.ts

```diff
@@ -1,23 +1,32 @@
-import { CreatorCollection, SourceCollection } from "~/types/search"
+import type { CreatorCollection, SourceCollection } from "~/types/search"
+
+const removeSlashes = (path: string): string => {
+  return path.replace(/^\/|\/$/g, "")
+}
```

Comment:
Does this only remove leading and trailing slashes from the path? What about trailing or leading slashes in creator names? e.g., `image/source/flickr//edgycreator//` where the creator name is `/edgycreator/`?

Reply:
@obulat have you seen this question? Just want to make sure that `path` gets passed in with the `/`s from the path, and not the raw creator name characters _on both ends of the string and always_.

If that _is_ the case, then can we just do `path.substring(1, path.length - 1)` and avoid the regex altogether? It would communicate things a bit more clearly, especially if the function is renamed to `trimLeadingAndTrailingSlashes` (or some less verbose version :grin: )

Reply:
I'm realising that potentially my question doesn't make sense anymore, because all slashes that are elements of the creator's name should be URL encoded, right? I am still waiting to approve just to make sure, because http://192.168.20.75:8443/audio/source/jamendo/creator/%2FThe.madpix.project%3F/, with a leading slash in the creator name URL encoded does work, but http://192.168.20.75:8443/audio/source/jamendo/creator//The.madpix.project%3F/ does not.

However, the PR description says that we should be able to visit http://localhost:8443/audio/source/wikimedia_audio/creator/wiki/Jeuwre/, but that doesn't work either, it 404s. The URL encoded version does work.

Except, they only work when navigate to client side. Navigating to them SSR (i.e., reloading the page) does not work. Is it an issue with preferences, perhaps? I also don't see the additional search view links on the single result page, so I think it may be an issue outside this PR, but I want to make absolutely sure it is working the right way, because what I am experiencing differs from the PR description.